core-web/libs/ui/src/lib/dot-tab-buttons/dot-tab-buttons.component.spec.ts

```diff
@@ -50,54 +56,107 @@ describe('DotTabButtonsComponent', () => {
 
     it('should emit openMenu event when showMenu is called', () => {
         const openMenuSpy = spyOn(spectator.component.openMenu, 'emit');
-        spectator.component.showMenu(null);
+
+        spectator.component.showMenu({
```

Comment:
We should never call the method on the component, if I got the HTML and remove the `(click)="showMenu"` this test will not fail but the functionality will break.